channeldb/graph.go

```diff
@@ -2018,13 +2044,19 @@ func delEdgeUpdateIndexEntry(edgesBucket kvdb.RwBucket, chanID uint64,
 }
 
 func delChannelEdge(edges, edgeIndex, chanIndex, zombieIndex,
-	nodes kvdb.RwBucket, chanID []byte, isZombie, strictZombie bool) error {
+	nodes kvdb.RwBucket, chanID []byte, isZombie, strictZombie bool,
+	graphCache *GraphCache) error {
```

Comment:
With this new parameter, maybe the function can be a method of the graph?

Reply:
Yes, done.